versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/platforms/_docker.mdx

```diff
@@ -3,7 +3,7 @@ import TabItem from '@theme/TabItem';
 import CodeBlock from '@theme/CodeBlock';
 import Link from '@docusaurus/Link';
 import styles from '@site/src/pages/index.module.css';
-import DockerFileGenerator from '@site/src/components/DockerFileGenerator/DockerFileLatest.jsx';
+import DockerFileGenerator from '@site/src/components/DockerFileGenerator/DockerFilePreRelease.jsx';
```

Comment:
This is not right is it? The pre-release Docker file in the version-latest docs?

Reply:
Thank you for catching this jim!